### PR TITLE
fix ping latency alert solution

### LINKED ISSUE #7383: https://docs.pingcap.com/zh/tidb/stable/alert-rules#blacker_ping_latency_more_than_1s
官方文档：
在 Grafana Blackbox Exporter dashboard 的 tcp 面板上检查是否有丢包。

实际：
TCP面板在 Node-exporter下

（文档中描述让客户产生质疑）

### PR DESCRIPTION
**Pull request by tabokie**
Signed-off-by: tabokie <xy.tao@outlook.com>

### What is changed, added or deleted? (Required)

Fix #7383

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

**Comment by TomShawn** [comment]
@tabokie Could you please invite a tech review? Thanks~

File: alert-rules.md [hunk 1]
```diff
@@ -1118,5 +1118,5 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 
 * 处理方法：
 
-    * 在 Grafana Blackbox Exporter dashboard 上检查两个节点间的 ping 延迟是否太高。
-    * 在 Grafana Blackbox Exporter dashboard 的 tcp 面板上检查是否有丢包。
+    * 在 Grafana Blackbox Exporter 页面上检查两个节点间的 ping 延迟是否太高。
+    * 在 Grafana Node Exporter 页面 的 TCP 面板上检查是否有丢包。
```
Comment (en-jin19) [inline alert-rules.md:1122]:
```suggestion
    * 在 Grafana Node Exporter 页面的 TCP 面板上检查是否有丢包。
```

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**